data.yml

```diff
@@ -578,6 +578,18 @@ people:
   - company: Stitch Fix
     position: Principal Engineer
     start_day: 2016-10-17
+    end_day: 2017-01-31
+  - company: Georgia General Assembly
+    position: House Representative for District 119
+    start_day: 2017-11-??
```

Comment:
maybe do 01 in here instead of ??. Not sure this is going to work the way you want!

Reply:
😆😆

This is all a placeholder so I can get the dates right and get the PR ready for a merge!